packages/datadog-plugin-mongodb-core/test/mongodb.spec.js

```diff
@@ -108,11 +108,16 @@ describe('Plugin', () => {
             }, () => {})
           })
 
-          it('should sanitize the query', done => {
+          it('should sanitize the query', function (done) {
+            this.timeout(1000 * 60)
```

Comment:
Why increase this that much?

Reply:
timeouts on my machine w/o the patch. with the patch it could be down at around 10s but it does take 10s to complete or so sometimes.

Reply:
We should never do any operation that could take that long in the first place, so there should be an upper limit that easily fits within Mocha's timeout.

Reply:
sounds fine